inc/cf7.php

```diff
@@ -45,10 +44,22 @@ function klypHsCf7CatchSubmission($result, $tags)
     $hsFormFields       = get_post_meta($cf7FormId, '_klyp-cf7-to-hubspot-hs-map-fields', true);
 
     // start hubspot
-    $hubspot = new klypHubspot();
+    $hubspot                = new klypHubspot();
     $hubspot->cf7FormId     = $cf7FormId;
     $hubspot->cf7FormFields = $cf7FormFields;
     $hubspot->hsFormFields  = $hsFormFields;
+
+    if ($files) {
+        $hsFilesPath = $hubspot->hsFileUpload($files);
+        if ($hsFilesPath) {
+        // merge the uploaded file of hubspot path with 
+            $mergeFilesWithFormData = array();
+            $mergeFilesWithFormData = array_merge($_POST, $hsFilesPath);
+            $_POST                  = $mergeFilesWithFormData;

```

Comment:
Im generally uneasy about assigning to the $_POST as it can break things down the line (if its messed with too much) 

maybe try

```
 if ($files) {
        $hsFilesPath = $hubspot->hsFileUpload($files);
        if ($hsFilesPath) {
            // merge the uploaded file of hubspot path with 
            $mergeFilesWithFormData = array_merge($_POST, $hsFilesPath);
            $hubspot->postedData    = klypCF7ToHubspotSanitizeInput($mergeFilesWithFormData);
        }
    } else {
        $hubspot->postedData    = klypCF7ToHubspotSanitizeInput($_POST);
    }

   
```